templates/sonar.properties.j2

```diff
@@ -34,4 +42,5 @@ ldap.user.emailAttribute={{ ldap.user_email_attribute | default('mail') }}
 # Group Configuration
 ldap.group.baseDn={{ ldap.group_base_dn | mandatory }}
 ldap.group.request={{ ldap.group_request | default('(&(objectClass=posixGroup)(memberUid={uid}))') }}
+ldap.group.idAttribute={{ ldap.group_idAttribute | default('cn') }}
 {%- endif %}
```

Comment:
Please add group_idAttribute line to defaults/main.yml also.

Reply:
Good finding! Totally forgot that. I changed and commited that.